### DISCUSSION TITLE
Not found Plots

### DISCUSSION THREAD

**Discussion by DimstrBarboss** [Chia-Network/chia-blockchain#8407] [Support]
Does not see part of the plots, I suspect that all the plots are made in the GUI. The sections made by Mad max work. This happened after a sudden restart of the computer. Newly made sections in the GUI are visible and working.

![Not found Plots](https://user-images.githubusercontent.com/90198409/132552395-708c10b2-6f17-44ca-9fe7-318d18976dfa.png)

**Comment by DimstrBarboss** [comment]
When checking for validity, it issues such a warning:
There are 1 plots with a farmer or pool public key that is not on this machine. The farmer private key must be in the keychain in order to farm them, use 'chia keys' to transfer keys. The pool public keys must be in the config.yaml

**Reply by loppefaaret** [reply]
and when you check what pool/farm public key it reports, does that show up if you do `chia keys show` ?

**Comment by DimstrBarboss** [comment]
Here is the data from the chia keys show:
Farmer public key (m/12381/8444/0/0): a8fc2b2bd8ddd-SNIP-96e53698737657b2810107
And here's what checking a non-working site gives:
Farmer public key: 9004eeda2a-SNIP-fd66fddae415a4dcdeb
Is there any way to fix this?

**Reply by DimstrBarboss** [reply]
This is the Farmer public key from my old mnemonic.
But how did it get into the rafts that I made in the new mnemonics?

**Reply by DimstrBarboss** [reply]
Adding the old mnemonics helped

**Comment by BrandtH22** [comment]
@DimstrBarboss , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
